ui-v2/src/App.js

```diff
@@ -1,16 +1,14 @@
-import React from 'react';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import { Spin } from 'antd';
-import { AuthProvider, PrivateRoute } from 'components';
-import './App.css';
+import React from "react";
```

Comment:
Can you revert all single-quote to double-quote fix?

Reply:
For all `*.js` ?

Reply:
Well, just the one that you unintentionally change should be fine. I believe you can just use the ESLint configuration to fix the quote.

Reply:
Done on the latest commit